18_reinforcement_learning.ipynb

```diff
@@ -1757,7 +1757,7 @@
     "        best_weights = model.get_weights() # Not shown\n",
     "        best_score = step # Not shown\n",
     "    print(\"\\rEpisode: {}, Steps: {}, eps: {:.3f}\".format(episode, step + 1, epsilon), end=\"\") # Not shown\n",
-    "    if episode > 50:\n",
+    "    if episode >= 50:\n",
```

Comment:
I know you didn't ask for clarifications here, but allow me to go ahead. The book says we start training after 50 episodes. That means we have to start at `episode = 50` (0-based), not `episode = 51` like previously.

Reply:
Yes, I guessed as much, good catch! 😊